src/gosling-track/gosling-track.ts

```diff
@@ -1002,11 +1010,13 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
                 const capturedElements = this.#getElementsWithinMouse(mouseX, mouseY);
 
                 if (capturedElements.length !== 0) {
-                    publish('click', {
-                        id: context.viewUid,
-                        genomicPosition,
-                        data: capturedElements.map(d => d.value)
-                    });
+                    this.options.siblingIds.forEach(id =>
+                        publish('click', {
+                            id,
+                            genomicPosition,
+                            data: capturedElements.map(d => d.value)
+                        })
+                    );
```

Comment:
My understanding is that this publishes an event for each `siblingId`. Does this happen for every sibling track as well, or just the top overlaid track? i.e. when an overlaid track gets clicked on, does every overlaid track within that view get clicked on, or just the top-most overlaid track?

Reply:
Yes, your understanding is correct. This just tries to ensure that we publish information for every track that can conceptually have an ID.

In many cases, users will be interested in a specific track and not all tracks, so we can improve the API further through hierarchical pubsub topics (https://github.com/mroderick/PubSubJS#hierarchical-addressing), e.g., 

```ts
// src/core/pubsub.ts
export function publishTrackSpecific<Name extends EventName>(name: Name, trackId, string, data: _EventMap[Name]): void {
    PubSub.publish(`${name}.${trackId}`, data); // e.g., 'click.track-1'
}

export function subscribeTrackSpecific<Name extends EventName>(
    name: Name,
    trackId: string
    callback: (msg: string, data: _EventMap[Name]) => void
): void {
    PubSub.subscribe(`${name}.${trackId}`, callback);
}

// `api.subscribe('click', fn)` subscribes to all tracks
// `api.subscribeTrackSpecific('click', 'track-1', fn)` subscribes to `track-1` only
```